mr_roboto.py

```diff
@@ -32,10 +33,19 @@ def on_modified(self, event):
             self.bot.reload(*local_plugins)
 
 
+def get_version():
+    regex = re.compile(r'VERSION = (.*)')
+    with open('Makefile') as mkfile:
+        for line in mkfile:
+            match = regex.match(line)
+            if match:
+                return match.group(1)
```

Comment:
Hmmm.. I think it would be better to abort (exit) execution in case there is no version or Makefile, what do you think? Or just set a default value and keep going?

Reply:
Got it. And I agree. I'm going to change it then.